index.html

```diff
@@ -107,168 +114,193 @@ <h2 class="work_primary_text">Multi-Post Stories</h2>
+                    </li>
 
-                    <div class="btn2-holder">
-                        See Project
-                    </div>
-                </div>
-            </div>
+                    <li class="grid-item6 item2">
+                        <div class="content">
+                            <div class="project1">
+                                <h3 class="p-text">Professional Art <br>
+                                    Printing Data
+                                </h3>
+                                <p class="s-text">
+                                    A daily selection of privately
+                                    personalized reads; no accounts or
+                                    sign-ups required. Has been the 
+                                    industry's standard.
+                                </p>
+                                <div class="tags">
+                                    <ul class="tag">
+                                        <li> <a href="#">HTML</a></li>
+                                        <li> <a href="#">Bootstrap</a></li>
+                                        <li> <a href="#">Ruby</a></li>
+                                    </ul>
+                                </div>
+                            </div>
+                            
+                            <div class="btn2-holder">
+                                See Project
+                            </div>
+                        </div>
+                    </li>
+                </ul>
+        </section>
         </div>
         <section class="main-container">
             <article class="about-container">
                 <article class="about">
-                        <h2 class="about-p-text" >About Me</h2>
+                    <article class="about-top">
+                        <h4 class="about-p-text" >About Me</h4>
                         <p class="about-s-text">Hello I’m a software developer! I can 
                             help you build a product , feature or 
                             website Look through some of my work 
                             and experience! If you like what you 
                             see and have a project you need coded, 
                             don’t hesitate to contact me.
                         </p>
-                        <div class="btn-holder">
+                        <div class="btn-holder resume-button">
                             <button class="btn">Get My Resume</button>
                         </div>
                         
-                        <img class="about-me-banner" src="./assets/images/About Me Background.svg" alt="Aboutmebanner">
-                        
+                    </article>
+                    <img class="about-me-banner" src="./assets/images/About Me Background.svg" alt="Aboutmebanner">
+                    <img class="about-me-desktop" src="./assets/images/aboutme-dekstop23.svg" alt="aboutmedesktop">
```

Comment:
- [ ] As requested by the previous reviewer, kindly use those as the background images for the whole `about` section.